frontend/src/features/mapSideBar/SidebarContents/subforms/IdentificationForm.tsx

```diff
@@ -0,0 +1,132 @@
+
+import { FastSelect, FastInput, SelectOptions, Check } from 'components/common/form';
+import { Label } from 'components/common/Label';
+import AddressForm from 'features/properties/components/forms/subforms/AddressForm';
+import InformationForm from 'features/properties/components/forms/subforms/InformationForm';
+import LatLongForm from 'features/properties/components/forms/subforms/LatLongForm';
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+import { senstiveTooltip as sensitiveTooltip } from '../../../../../src/features/properties/components/forms/strings';
+import { HARMFUL_DISCLOSURE_URL } from 'constants/strings';
+
+interface IIdentificationProps {
+  agencies: SelectOptions;
+  classifications: SelectOptions;
+  predominateUses: SelectOptions;
+  constructionType: SelectOptions;
+  formikProps: any;
+  nameSpace?: any;
+  setMovingPinNameSpace: (nameSpace: string) => void;
+}
+
+export const IdentificationForm: React.FC<IIdentificationProps> = ({
+  formikProps,
+  agencies,
+  classifications,
+  predominateUses,
+  constructionType,
+  nameSpace,
+  setMovingPinNameSpace,
+}) => {
+  const withNameSpace: Function = React.useCallback(
+    (name?: string) => {
+      return [nameSpace ?? '', name].filter(x => x).join('.');
+    },
+    [nameSpace],
+  );
+  return (
+    <Container>
+      <Row>
+        <h4>Building Identification</h4>
+      </Row>
+      <Row>
+        <Col>
+          <InformationForm
+            isAdmin
+            wizard
+            agencies={agencies}
+            classifications={classifications}
+            nameSpace={withNameSpace('')}
+          />
+        </Col>
+        <Col>
+          <Row>
+            <Label required>Main Usage</Label>
+            <FastSelect
+              formikProps={formikProps}
+              // disabled={disabled || readonly}
+              placeholder="Must Select One"
+              field={withNameSpace('buildingPredominateUseId')}
+              type="number"
+              options={predominateUses}
+            />
+          </Row>
+          <Row>
+            <Label required>Construction Type</Label>
+            <FastSelect
+              formikProps={formikProps}
+              // disabled={props.disabled || readonly}
+              placeholder="Must Select One"
+              field={withNameSpace('buildingConstructionTypeId')}
+              type="number"
+              options={constructionType}
+            />
+          </Row>
+          <Row>
+            <Label>Number of Floors</Label>
+            <FastInput
+              displayErrorTooltips
+              className="input-small"
+              formikProps={formikProps}
+              field={withNameSpace('buildingFloorCount')}
+              type="number"
+            />
+          </Row>
+          <Row>
+            <Label>SPP</Label>
+            <FastInput formikProps={formikProps} field="spp" />
```

Comment:
We have a spp field now that isn't controlled by claims?

Reply:
What is the `spp` field?  Is this the project number?  If so review the latest `dev` branch to see how it should be implemented.